pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/errorprone/xml/OverrideBothEqualsAndHashcode.xml

```diff
@@ -210,4 +210,16 @@ public class Foo {
             }
             ]]></code>
     </test-code>
+    <test-code>
+        <description>No feature envy on MissingEqualsOnComparableRule: Comparable without equals</description>
```

Comment:
ignores comparable to avoid collision.

https://github.com/pmd/pmd/pull/5856#discussion_r2203503403